bazel/repository_locations.bzl

```diff
@@ -475,7 +476,8 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         sha256 = "5f156edb65d20b856d638dd9ee2dfb43285914d9aa2b6ec779dac0270cd56c3f",
         strip_prefix = "ragel-{version}",
         urls = ["https://www.colm.net/files/ragel/ragel-{version}.tar.gz"],
-        use_category = ["controlplane", "dataplane_core"],
+        use_category = ["controlplane", "dataplane_ext"],
```

Comment:
Thanks. Why is it `controlplane`? I think this could just be `dataplane_ext`.

Reply:
RE2 is used in a bunch of places, e.g. admin config dump handler. We only have pluggable matchers for things like route matching which occurs on the data plane. It might be possible to argue that we are instantiating the regex matcher at config time, but I would err towards considering this a dataplane_ext only. If this doesn't work, we probably should add a controlplane_ext category.

Reply:
I think the change of use category does not affect codes, so it doesn't matter.